index.js

```diff
@@ -121,6 +152,26 @@ function handleArguments(env) {
     exit(1);
   }
 
+  customizeConfig(['home', 'project', 'cwd'].map(getConfigFile));
```

Comment:
The purpose of this code is to pass config. properties to `customizeConfig` function in the order of the keys `home`, `project` and `cwd`, because object properties order is not guaranteed.